WhateverGreen/kern_rad.cpp

```diff
@@ -409,6 +412,12 @@ void RAD::processHardwareKext(KernelPatcher &patcher, size_t hwIndex, mach_vm_ad
 			patcher.clearError();
 		}
 	}
+
+	// Patch AppleGVA support for non-supported models
+	if (forceCodecInfo) {
+		KernelPatcher::RouteRequest request(getHWInfoProcNames[hwIndex], wrapGetHWInfo[hwIndex], orgGetHWInfo[hwIndex]);
```

Comment:
I believe this one is missing a `&& getHWInfoProcNames[hwIndex] != nullptr` check.